Formula/silk.rb

```diff
@@ -1,8 +1,9 @@
 class Silk < Formula
   desc "Collection of traffic analysis tools"
   homepage "https://tools.netsa.cert.org/silk/"
-  url "https://tools.netsa.cert.org/releases/silk-3.19.1.tar.gz"
-  sha256 "b287de07502c53d51e9ccdcc17a46d8a4d7a59db9e5ae7add7b82458a9da45a7"
+  url "https://tools.netsa.cert.org/releases/silk-3.19.2.tar.gz"
+  sha256 "358ba718208dcfb14a22664a6d935f343bd7a1976729e5619ba7c702b70e3a7d"
+  license "GPL-2.0-or-later"
```

Comment:
Should this be `GPL-2.0-only`? Ref: https://tools.netsa.cert.org/silk/license.html

Reply:
Sorry I should have explained why I put that first. I looked trough the source code in the .tar.gz and it also includes other librairies such as `libflowsource` which is `GPL-2.0-or-later` (it include the "or (at your option) any later version" part).

Reply:
Sorry for the delayed response. Does this mean we need to include multiple licenses with `all_of` and specify through comments the paths to which specific licenses apply? (e.g. `e2fsprogs`)

I wonder whether `GPL-2.0-or-later` prevails over the `GPL-2.0-only` of the library itself. Anyhow, I'm not too familiar with licensing, so if you think this is fine as-is, we can merge.

Reply:
Looking at other formulae, it does seems we would need an `all_of`. Since I'm sure if `silk` itself, without its dependencies use `GPL-2.0-or-later` or `GPL-2.0-only`, I postpone this to another PR.